tasks/gov.js

```diff
@@ -147,47 +100,123 @@ task("gov:setChainlinkPriceFeed")
             fork = await setupGovernanceFork("governor");
         }
         const et = require("../test/lib/eTestLib");
-        const ctx = isfork ? fork.ctx: await et.getTaskCtx();
+        const ctx = isfork ? fork.ctx : await et.getTaskCtx();
 
-        const underlying = isfork? args.underlying : await et.taskUtils.lookupToken(ctx, args.underlying);
+        const underlying = await et.taskUtils.lookupToken(ctx, args.underlying);
 
         if (isfork) {
             const admin = fork.admin;
-            await et.taskUtils.runTx(ctx.contracts.governance.connect(admin).setChainlinkPriceFeed(underlying, args.chainlinkAggregator, await ctx.txOpts()));
+            await et.taskUtils.runTx(ctx.contracts.governance.connect(admin).setChainlinkPriceFeed(underlying.address, args.chainlinkAggregator, await ctx.txOpts()));
         } else {
             await et.taskUtils.runTx(ctx.contracts.governance.setChainlinkPriceFeed(underlying.address, args.chainlinkAggregator, await ctx.txOpts()));
         }
     });
 
+
+
+task("gov:forkAccountsAndHealthScores", "Get all unique accounts that have entered an Euler market and their health scores")
+    .addPositionalParam("filename", "file name without .json suffix")
+    .setAction(async ({ filename }) => {
+        const fs = require("fs");
+
+        const et = require("../test/lib/eTestLib");
+        const { ctx, } = await setupGovernanceFork();
+
+        const transactions = await ctx.contracts.euler.queryFilter(
+            "EnterMarket",
+            "earliest",
+            "latest",
+        );
+
+        let result = [];
+        for (let i = 0; i < transactions.length; i++) {
+            let temp = {};
+            temp.account = transactions[i].args.account;
+            temp.underlying = transactions[i].args.underlying;
+            result.push(temp);
+        }
+
+        // unique event arguments based on accounts
+        const key = 'account';
+        const arrayUniqueByKey = [...new Map(result.map(item =>
+            [item[key], item])).values()];
+
+        // unique addresses regardless of markets 
+        const uniqueAddresses = [...new Set(arrayUniqueByKey.map(item => item.account))];
+
+        // compute health scores
+        let health_scores = {};
+        
+        console.log(`Number of unique addresses to parse in batches: ${uniqueAddresses.length}`);
+        
+        while (uniqueAddresses.length > 0) {
+            console.log("=== Waiting for the next batch to be resolved")
```

Comment:
it'd be good to display how many addresses left to be processed

Reply:
Will add now @kasperpawlowski thanks!